app/views/tutorials/show.html.erb

```diff
@@ -1,70 +1,75 @@
 <main class="tutorials">
 <h2><%= @facade.title %></h2>
 <h1 id="message"></h1>
-<div class="col col-4">
-  <h4>Videos</h4>
-  <ul>
-    <% @facade.videos.each do |video| %>
-      <li>
-        <h3><%= link_to video.title, tutorial_path(video_id: video.id), class: "show-link video-link", id: video.position %></h3>
-      </li>
-    <% end %>
-  </ul>
-</div>
-
-<div class="col col-8">
-  <div class="title-bookmark">
-    <h3><%= @facade.current_video.title %></h3>
-    <div class="bookmarks-btn">
-      <% if current_user %>
-      <%= button_to "Bookmark", user_videos_path(video_id: @facade.current_video.id, user_id: current_user.id), method: :post, class: "btn btn-outline mb1 teal" %>
-      <% else %>
-      <p class="btn btn-outline mb1 teal" %> Bookmark </p>
-      <p><i>User must login to bookmark videos</i></p>
-      <% end %>
+  <%if @facade.videos.empty? %>
+    <p><i>There are no videos in this tutorial</i></p>
```

Comment:
Currently you can create a tutorial with no videos in it. that caused a few things on this page to break